packages/react/src/auth/utils.ts

```diff
@@ -0,0 +1,9 @@
+import type { GadgetSession } from "./useSession";
+
+export const isSessionSignedOut = (session: GadgetSession) => {
+  return session?.userId == undefined;
```

Comment:
Could `userId` be any other falsy values? I'm thinking maybe it's better to be `!!session?.userId`